sphinxcontrib/needs/needsfile.py

```diff
@@ -131,7 +132,12 @@ def load_json(self, file):
             self.log.debug(f"needs.json file loaded: {file}")
 
 
-def check_needs_file(path):
+class Errors:
+    def __init__(self, schema_errors: List[Any]):
```

Comment:
@danwos what is the 'type' of the schema errors? I'd prefer to avoid using `Any` if i can